documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -6,48 +6,118 @@
+You can use the name of the `KafkaRebalance` resource to return a summary from the command line.
 
-[[contents-optimization-proposals]]
-[discrete]
-== Contents of optimization proposals
+.Returning an optimization proposal summary
+[source,shell,subs=+quotes]
+----
+kubectl describe kafkarebalance _<kafka_rebalance_resource_name>_ -n _<namespace>_
+----
+
+You can also use the `jq` command line JSON parser tool.
+
+.Returning an optimization proposal summary using jq
+[source,shell,subs=+quotes]
+----
+`kubectl get kafkarebalance -o json | jq _<jq_query>_`.
+----
+
+Use the summary to decide whether to approve or reject an optimization proposal.
 
-An optimization proposal consists of two main sections:
+Approving an optimization proposal:: You approve the optimization proposal by setting the `strimzi.io/rebalance` annotation of the `KafkaRebalance` resource to `approve`.
+Cruise Control applies the proposal to the Kafka cluster and starts a cluster rebalance operation.
+Rejecting an optimization proposal:: If you choose not to approve an optimization proposal,
+you can xref:proc-generating-optimization-proposals-str[change the optimization goals] or xref:rebalance_tuning_options[update any of the rebalance performance tuning options], and then generate another proposal.
+You can use the `strimzi.io/refresh` annotation to generate a new optimization proposal for a `KafkaRebalance` resource.
 
-* The _summary_ is stored in the `status` of the `KafkaRebalance` resource.
-* The _broker load_ is stored in a ConfigMap that contains data as a JSON string.
+Use optimization proposals to assess the movements required for a rebalance.
+For example, a summary describes inter-broker and intra-broker movements.
+Inter-broker rebalancing moves data between separate brokers.
+Intra-broker rebalancing moves data between disks on the same broker when you are using a JBOD storage configuration.
+Such information can be useful even if you don't go ahead and approve the proposal.
 
-The summary provides an overview of the proposed cluster rebalance and indicates the scale of the changes involved.
-The broker load shows before and after values for the proposed rebalance, so you can see the impact on each of the brokers in the cluster.
+You might reject an optimization proposal, or delay its approval, because of the additional load on a Kafka cluster when rebalancing.
+
+In the following example, the proposal suggests the rebalancing of data between separate brokers.
+The rebalance involves the movement of 55 partition replicas, totaling 12MB of data, across the brokers.
+Though the inter-broker movement of partition replicas has a high impact on performance, the total amount of data is not large.
+If the total data was much larger, you could reject the proposal, or time when to approve the rebalance to limit the impact on the performance of the Kafka cluster.
+
+Rebalance performance tuning options can help reduce the impact of data movement.
```

Comment:
Can we link to the docs that talk about configuring the rebalance performance tuning options?

Reply:
We provide a link on line 54. Do we want another?